windows/dummy.go

```diff
@@ -0,0 +1,3 @@
+package windows
+
+/* Placeholder so `go get` doesn't complain about a lack of files */
```

Comment:
Like it says on the tin. Go get pulls down master and throws an error if there aren't any buildable files for the given platform.